### DISCUSSION TITLE
Rainbow Kit's business model

### DISCUSSION THREAD

**Discussion by MoritzKa** [rainbow-me/rainbowkit#1605] [Q&A] [answered]
Rainbow kit is a great project. Hope it continues and is sustainable. How is Rainbow going to earn money with it? 
Are you planning to charge wallets, apps or end-users?

**Comment by DanielSinclair** [comment, marked as answer]
@MoritzKa RainbowKit is an open-source public good akin to projects like Wagmi. Rainbow is a company that builds wallets, and RainbowKit is our way of improving the UX across the dApp ecosystem for our users and beyond. There are no plans to monetize this "Connect Wallet" functionality.
